package exceptiondemo;
import java.util.*;

public class ExceptionDemo {

    
    public static void main(String[] args) {
        int c;
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter two numbers");
        int a=sc.nextInt();
        int b=sc.nextInt();
        try
        {
            c=a/b;
             System.out.println(c);
        }
        catch(ArithmeticException e)
        {
            System.out.println("The number is not divisible by zero");
        }
       System.out.println("Bye");
        
        
        
        
        
        
        
    }
    
}
